system/ee/ExpressionEngine/Addons/forum/mod.forum_core.php

```diff
@@ -3305,7 +3305,7 @@ public function thread_rows($data, $is_announcement = false, $thread_review = fa
                         $temp,
                         array(
                             'lang:change_status' => ($row['status'] == 'o') ? lang('close_thread') : lang('activate_thread'),
-                            'path:change_status' => ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . ee()->functions->fetch_action_id('Forum', 'change_status') . '&amp;topic_id=' . $row['post_id'] . '&amp;board_id=' . $this->fetch_pref('board_id') . '&amp;trigger=' . $this->trigger,
+                            'path:change_status' => ee()->functions->fetch_site_index(0, 0) . QUERY_MARKER . 'ACT=' . ee()->functions->fetch_action_id('Forum', 'change_status') . '&amp;topic_id=' . $row['post_id'] . '&amp;board_id=' . $this->fetch_pref('board_id') . '&amp;trigger=' . $this->trigger .  '&amp;csrf_token=' . CSRF_TOKEN,
```

Comment:
This does not seem to respect `disable_csrf_protection` config setting.

Besides that, we really should avoid passing CSRF token in GET request. It is supposed to be sent as header, so I guess you can just use that